app/components/elements/Select.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Flex } from 'rebass/styled-components';
+import { Select as Base, Label, SelectProps } from '@rebass/forms';
+import styled from 'styled-components';
+import cx from 'classnames';
+import map from 'lodash/map';
+import KeyboardArrowDownRoundedIcon from '@material-ui/icons/KeyboardArrowDownRounded';
+
+import { borders, colors, radii, space } from '../../themes/baseTheme';
+import { Caption } from './FontStyles';
+
+const StyledSelect = styled(Flex)`
+  position: relative;
+  flex-wrap: wrap;
+
+  > div {
+    flex-grow: 1;
+  }
+
+  /* Override browser default styles for selects */
+  select {
+    appearance: none;
+    border: none;
+    padding: ${space[2]}px;
+  }
+
+  /* Hide the default dropdown icon */
+  select + svg {
+    display: none;
+  }
+
+  .MuiSvgIcon-root {
+    position: absolute;
+    right: ${space[2]}px;
+    color: inherit;
+    /* Disable pointer events so click actually applies to the dropdown menu underneath */
+    pointer-events: none;
+  }
+
+  &.disabled {
+    color: ${colors.text.primaryDisabled};
+    border-color: ${colors.lightestGrey};
+    background-color: ${colors.lightestGrey};
+  }
+`;
+
+export const Select = props => {
+  const { disabled, name, label, value, options, onChange, ...selectProps } = props;
+  const classNames = cx({ disabled });
+
+  return (
+    <React.Fragment>
+      {label && (
+        <Label htmlFor={name}>
+          <Caption>{label}</Caption>
+        </Label>
+      )}
+      <StyledSelect alignItems="center" className={classNames} {...selectProps}>
+        <Base
+          id={name}
+          name={name}
+          disabled={disabled}
+          value={value}
+          onChange={onChange}
+        >
+          {map(options, option => (
+            <option
+              id={option.value}
+              key={option.value}
+              value={option.value}
+            >
+              {option.label}
+            </option>
+          ))}
+        </Base>
+        <KeyboardArrowDownRoundedIcon />
+      </StyledSelect>
+    </React.Fragment>
+  );
+};
+
+Select.propTypes = {
+  ...SelectProps,
+  name: PropTypes.string.isRequired,
+  label: PropTypes.string,
+  disabled: PropTypes.bool,
+  value: PropTypes.string,
+  onChange: PropTypes.func.isRequired,
+  options: PropTypes.arrayOf(PropTypes.shape({
+    value: PropTypes.string.isRequired,
+    label: PropTypes.string.isRequired,
+  })).isRequired,
+};
+
+Select.defaultProps = {
+  width: ['100%', '75%', '50%'],
+  color: colors.text.primary,
+  sx: {
```

Comment:
It looks like `defaultProps` doesn't do a sort of _.extend or _.merge, so if we provide a value for `sx` that doesn't include _all_ of these object properties, then the remaining ones are lost. We may need to break these out?

Reply:
I agree.  I took a better approach on the Icons that allowed easy overridding of the default rebass styles.  I'll do the same here.

Reply:
I've moved the default styles to a new `inputs.select` variant.

There are a number of styles consistent with the `TextInput` component, and I'm wondering if it's worth migrating the `TextInput` styling over to an `inputs.text` variant, which will afford it the same flexibility we're gaining here.

Reply:
It probably is. Once these get merged in, I'll likely do that as part of the search component.